src/components/ReplSection.js

```diff
@@ -0,0 +1,105 @@
+import React from "react"
+import {
+  AspectRatio,
+  Box,
+  Container,
+  Heading,
+  Text,
+  SimpleGrid,
+  Stack,
+} from "@chakra-ui/react"
+
+import { Prism as SyntaxHighlighter } from "react-syntax-highlighter"
+import { a11yDark } from "react-syntax-highlighter/dist/cjs/styles/prism"
+
+//TODO: We could remove xarrray installation step by requesting xarray to be added to the Pyodide standard lib
+//TODO: Or wait until jupyterlite has streamlined the installation process of user-specified packages
```

Comment:
I think this is useful enough to warrant the small inconvenience of running micropip here.